COMETwebapp/COMETwebapp/Program.cs

```diff
@@ -22,14 +22,27 @@
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
+using Blazored.Modal;
+using CDP4Dal;
 using COMETwebapp;
+using COMETwebapp.SessionManagement;
+using Microsoft.AspNetCore.Components.Authorization;
 using Microsoft.AspNetCore.Components.Web;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 
 var builder = WebAssemblyHostBuilder.CreateDefault(args);
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
-builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddBlazoredModal();
+builder.Services.AddScoped(sp => new HttpClient());
```

Comment:
I don't think we need to register the HttpClient as this is wrapped by the ISession. I think this can be removed.